dvc/dvcfile.py

```diff
@@ -122,7 +123,7 @@ def _check_gitignored(self):
         if self._is_git_ignored():
             raise FileIsGitIgnored(self.path)
 
-    def _load(self):
+    def _load(self, use_pydantic: bool = False):
```

Comment:
To test new schema, you can toggle this to `True`, and it should start using the new schema for validation.